core/src/main/scala/org/ensime/core/Analyzer.scala

```diff
@@ -151,6 +155,7 @@ class Analyzer(
 
     case FullTypeCheckCompleteEvent =>
       broadcaster ! FullTypeCheckCompleteEvent
+      context.parent ! FullTypeCheckCompleteEvent
```

Comment:
It does. But I needed to use this in `AnalyzerManager` to handle the requests which didn't have a module to them.